docs/purchasing/parts.json

```diff
@@ -134,5 +134,90 @@
         "url": "https://www.amazon.com/Inch-XHF-Waterproof-Insulation-Wear-Resistant/dp/B0919F4YFZ"
       }
     ]
+  },
+  {
+    "pn": "ec-microfit-f-wire-1x3",
+    "notes": "Micro-Fit 3.0 Female Wire Housing 1x3",
+    "manufacturer": "Molex",
+    "mpn": "0436450300",
+    "avg_price": 0.49,
+    "price_unit": "",
+    "sources": [
+      {
+        "supplier": "Digikey",
+        "spn": "WM1846-ND",
+        "price": 0.49,
+        "price_units": 1,
+        "url": "https://www.digikey.com/en/products/detail/molex/0436450300/268975"
+      }
+    ]
+  },
+  {
+    "pn": "ec-microfit-f-crimp-20-24awg",
+    "notes": "Molex Micro-Fit 3.0 Female Crimp Contact 20-24awg",
+    "manufacturer": "Molex",
+    "mpn": "43030-0002",
+    "avg_price": 0.09,
+    "price_unit": "",
+    "sources": [
+      {
+        "supplier": "DigiKey",
+        "spn": "WM1125CT-ND",
+        "price": 0.09,
+        "price_units": 1,
+        "url": "https://www.digikey.com/en/products/detail/molex/0430300002/467797"
+      }
+    ]
+  },
+  {
+    "pn": "ec-kk-f-wire-3",
+    "notes": "Molex KK 2.54 Female Wire Housing 3 pos",
+    "manufacturer": "Molex",
+    "mpn": "22-01-3037",
+    "avg_price": 0.12,
+    "price_unit": "",
+    "sources": [
+      {
+        "supplier": "Mouser",
+        "spn": "538-22-01-3037",
+        "price": 0.12,
+        "price_units": 1,
+        "url": "https://www.mouser.com/ProductDetail/Molex/22-01-3037"
+      }
+    ]
+  },
+  {
+    "pn": "ec-kk-f-crimp",
+    "notes": "Molex KK 2.54 Female Crimp Contact 22-30awg",
+    "manufacturer": "Molex",
+    "mpn": "08-55-0102",
+    "avg_price": 0.10,
+    "price_unit": "",
+    "sources": [
+      {
+        "supplier": "Mouser",
+        "spn": "538-08-55-0102",
+        "price": 0.10,
+        "price_units": 1,
+        "url": "https://www.mouser.com/ProductDetail/Molex/08-55-0102"
+      }
+    ]
+  },
+  {
+    "pn": "ew-26awg-kit",
+    "notes": "10-color 24awg Wire Kit",
+    "manufacturer": "Remington",
+    "mpn": "26UL1007STRKIT10COLOR",
+    "avg_price": 42.35,
+    "price_unit": "ft",
```

Comment:
I think I may have partially fixed this. This field actually names the units. Would prefer metric here as likely the documentation will use that too.
The "price_units" field in a "supplier" listing below are then a denominator in these units listed here for how big of a batch is being sold.
I see how this could be cause for confusion and I'm soliciting ideas for how to name these more intuitively.

Reply:
Ok, so this is a bit nitpickty, but I recommend "price_unit" to be "m", which would change the "price_units" below to be converted, and then "avg_price" above would be divided out to be the per meter price. At the very least it is definitely not the case that price per foot is 106.64USD as it appears now.
See definition for "ew-shrink5" above as example.

Again, I concede that "price_unit" and "price_units" is confusing, so if you have suggestions for how we should name these to be more intuitive, I will make a PR to change the convention.